src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+* {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  color: #060b35;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198.4px;
+  height: 406px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  flex-direction: column;
+  transition-duration: $hover-duration;
```

Comment:
It looks like `$hover-duration` variable is used without being defined in this file. Make sure that the variables are imported from the `variables.scss` file to ensure they are available in this context.